__tests__/app.test.js

```diff
@@ -295,3 +295,5 @@ describe("POST /api/articles/:article_id/comments", () => {
       });
   });
 });
+
```

Comment:
line 276, your error message is about the URL, when the error is in the bad request: suggest changing that